### PR TITLE
Video: rename variables, cleanup

### PR DESCRIPTION
**Pull request by danirabbit**
* Name these properties closer to the conventions of GLib.File and Gst.Tags
* Simplify object construction

### CODE REVIEW

File: src/Objects/Video.vala [hunk 1]
```diff
@@ -20,50 +20,51 @@
...
-        if (directory != Environment.get_user_special_dir (UserDirectory.VIDEOS)) {
-            container = Path.get_basename (directory);
+        if (path != Environment.get_user_special_dir (UserDirectory.VIDEOS)) {
+            show_name = Path.get_basename (path);
         }
 
-        hash_file_poster = GLib.Checksum.compute_for_string (ChecksumType.MD5, video_file.get_uri (), video_file.get_uri ().length);
-        hash_episode_poster = GLib.Checksum.compute_for_string (ChecksumType.MD5, video_file.get_parent ().get_uri (), video_file.get_parent ().get_uri ().length);
+        hash_file_poster = GLib.Checksum.compute_for_string (ChecksumType.MD5, file.get_uri (), file.get_uri ().length);
+        hash_episode_poster = GLib.Checksum.compute_for_string (ChecksumType.MD5, file.get_parent ().get_uri (), file.get_parent ().get_uri ().length);
```
Comment (tintou) [inline src/Objects/Video.vala:67]: No need to specify the name here:
```suggestion
        hash_file_poster = GLib.Checksum.compute_for_string (ChecksumType.MD5, file.get_uri ());
        hash_episode_poster = GLib.Checksum.compute_for_string (ChecksumType.MD5, file.get_parent ().get_uri ());
```

File: src/Objects/Video.vala [hunk 2]
```diff
@@ -20,50 +20,51 @@
...
 
-    public File video_file { get; private set; }
     public bool poster_initialized { get; private set; default = false; }
+    public File file { get; private set; }
     public Gdk.Pixbuf? poster { get; private set; default = null; }
-    public string container { get; private set; default = ""; }
     public string hash_episode_poster { get; private set; }
     public string hash_file_poster { get; private set; }
     public string poster_cache_file { get; private set; }
+    public string show_name { get; private set; default = ""; }
```
Comment (tintou) [inline src/Objects/Video.vala:34]: Maybe more?
```suggestion
    public string display_name { get; private set; default = ""; }
```

Comment (danirabbit) [reply src/Objects/Video.vala:34]: This is actually meant to be the equivalent of Gst.Tags.SHOW_NAME which is the name of three tv show or series that an episode belongs to

**Comment by jeremypw** [comment]
@danirabbit Please resolve conflicts.

**Comment by danirabbit** [comment]
Oh yeah I think this is just obsolete now. Thanks for bumping!

**Pull request closed without merging**